std/experimental/logger/core.d

```diff
@@ -1631,7 +1631,7 @@ private @property Logger defaultSharedLoggerImpl() @trusted
     import std.conv : emplace;
     import std.stdio : stderr;
 
-    static __gshared ubyte[__traits(classInstanceSize, FileLogger)] _buffer;
+    static __gshared align(FileLogger.alignof) void[__traits(classInstanceSize, FileLogger)] _buffer;
```

Comment:
@ZombineDev that'd be great, thanks. There seem to be three kinds - thread-local, shared, and gshared.